caller-service/skaffold.yaml

```
@@ -10,12 +10,17 @@ build:
       env:
         - BP_BOOT_NATIVE_IMAGE=true
 deploy:
-  kubectl:
-    manifests:
-      - k8s/ksvc.yaml
+  kubectl: {}
+manifests:
+  rawYaml:
+    - k8s/ksvc.yaml
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)